content/v2.0/reference/flux/stdlib/experimental/join.md

```diff
@@ -0,0 +1,161 @@
+---
+title: experimental.join() function
+description: >
+  The `experimental.join()` function joins two streams of tables on the
+  group key with the addition of the `_time` column.
+menu:
+  v2_0_ref:
+    name: experimental.join
+    parent: Experimental
+weight: 302
+---
+
+The `experimental.join()` function joins two streams of tables on the
+[group key](/v2.0/reference/glossary/#group-key) and `_time` column.
+Columns that are not part of the group key or explicitly mapped in the join operation
+are dropped from output tables.
```

Comment:
`experimental.join` will never implicitly drop columns. Whichever fields the user specifies in the `fn` argument, only those columns will be output. A user can return only a subset of the group key columns if desired.